tests/libcxx/contest.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+---
+  tests-root: '..\..\llvm-project\libcxx\test\std'

```

Comment:
is this a "unique to stl" format, or is it a general contest thing.

If it's unique to us then having paths be relative to the git repository root would be cool.

Reply:
This is specific to the Microsoft-internal "Contest" distributed test harness's "libcxx runner". A file outside of the GitHub repo tells Contest to look at `tests/libcxx/contest.yaml`, which then says to look at the `llvm-project` submodule's tests. Contest doesn't specifically know about the location of the GitHub repo root.

This new `tests-root` variable achieves the important cleanup of simplifying all of the paths in `skipped_tests.txt`; it is unclear what further simplification is possible.

Reply:
OK. In that case this relative path is fine.